Animations/Animation.js

```diff
@@ -107,14 +108,14 @@ var makeTransition = function (displayName, transitions) {
       return (
         React.DOM.div(
           {
-            className: displayName,
+            className: className,
             key: this.props.key
```

Comment:
I'm not sure if you can directly access the `key` property from `this.props` in `Reactv0.12.2`.  I think the has to be accessed throw an internal property.  Maybe something like `this._key`.  I'm not certain however.  At the same time, do we necessarily need to pass in a key here?  If this is just rendering a div than could we use jsx?  Since the `makeAnimation` function just returns a single React component and not an array it shouldn't need to pass in a `key`.